### PR TITLE
watchguard_firebox_mr

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5098

### CODE REVIEW

File: Packs/WatchguardFirebox/ModelingRules/WatchguardFirebox_1_3/WatchguardFirebox_1_3.xif [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[MODEL: dataset="watchguard_xtm_raw"]
+alter proto = uppercase(proto)
+| alter xdm.event.id = leefEventID,
+    xdm.observer.version = leefProductVersion,
+    xdm.source.port = to_integer(srcPort),
```
Comment (evisochek) [inline Packs/WatchguardFirebox/ModelingRules/WatchguardFirebox_1_3/WatchguardFirebox_1_3.xif:5]: @guytamir10 Isn't this field INT already?

Comment (guytamir10) [reply Packs/WatchguardFirebox/ModelingRules/WatchguardFirebox_1_3/WatchguardFirebox_1_3.xif]: was fixed

File: Packs/WatchguardFirebox/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+# Watchguard Firebox
+This pack includes Cortex XSIAM content
+
+# How to config to send syslog LEEF logs
+
+1. Select System > Logging.
```
Comment (ShirleyDenkberg) [inline Packs/WatchguardFirebox/README.md:7]:
```suggestion
1. Select **System** > **Logging**.
```

File: Packs/WatchguardFirebox/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   The Logging page appears.
+2. Select the Syslog Server tab.
+3. Select the Send log messages to the syslog server at this IP address check box.
```
Comment (ShirleyDenkberg) [inline Packs/WatchguardFirebox/README.md:10]:
```suggestion
3. Click the **Send log messages to the syslog server at this IP address** checkbox.
```

File: Packs/WatchguardFirebox/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+4. In the IP Address text box, type the IP address for the syslog.
+5. In the Port text box, the default syslog server port (514) appears. To change the server port, type or select a different port for your server.
+6. From the Log Format drop-down list, select IBM LEEF.
```
Comment (ShirleyDenkberg) [inline Packs/WatchguardFirebox/README.md:13]:
```suggestion
6. From the **Log Format** drop-down list, select **IBM LEEF**.
```

File: Packs/WatchguardFirebox/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+# Watchguard Firebox
+This pack includes Cortex XSIAM content
+
+# How to config to send syslog LEEF logs
+
+1. Select System > Logging.
+   The Logging page appears.
+2. Select the Syslog Server tab.
+3. Select the Send log messages to the syslog server at this IP address check box.
+4. In the IP Address text box, type the IP address for the syslog.
```
Comment (ShirleyDenkberg) [inline Packs/WatchguardFirebox/README.md:11]:
```suggestion
4. In the **IP Address** text box, type the IP address for the syslog.
```

File: Packs/WatchguardFirebox/README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+5. In the Port text box, the default syslog server port (514) appears. To change the server port, type or select a different port for your server.
+6. From the Log Format drop-down list, select IBM LEEF.
+
+![Server Screenshot](./doc_imgs/syslogleef.png)
+
+8. In the Syslog Settings section, for each type of log message, select a syslog facility from the drop-down list.
+   If you select the IBM LEEF log format, you must select the The syslog header check box before you can select the syslog facility for the log message types.
```
Comment (ShirleyDenkberg) [inline Packs/WatchguardFirebox/README.md:18]:
```suggestion
   If you select the IBM LEEF log format, you must select the syslog header checkbox before you can select the syslog facility for the log message types.
```

File: Packs/WatchguardFirebox/README.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+   For high-priority syslog messages, such as alarms, select Local0.
+   To assign priorities for other types of log messages (lower numbers have greater priority), select Local1–Local7.
```
Comment (ShirleyDenkberg) [inline Packs/WatchguardFirebox/README.md:20]:
```suggestion
   To assign priorities for other types of log messages (lower numbers have greater priority), select **Local1–Local7**.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**